### PR TITLE
st1 [2014][ADD] stock_picking_ready_flag

### PR DESCRIPTION
**Pull request by tack-yabeQuartile**
[#2014](https://www.quartile.co/web#id=2014&model=project.task&view_type=form&menu_id=)

* Add picking_flag to control picking.
* Add a filter to the list view of stock.picking to display only records that have been allowed to ship.

This PR supersedes [rpl-private 225.](https://github.com/qrtl/rpl-private/pull/225)

### CODE REVIEW

File: stock_picking_ready_flag/models/__init__.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+from . import stock_picking
+from . import picking_type
```
Comment (TomohiroAbe-Quartile) [inline stock_picking_ready_flag/models/__init__.py]: Its a tiny change, sort in ABC order.

File: stock_picking_ready_flag/views/stock_picking_views.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='scheduled_date']" position="before">
+                <field
+                    name="is_ready"
+                    attrs="{'invisible':[('picking_type_code','!=','outgoing')]}"
+                />
+            </xpath>
+        </field>
+    </record>
+    <record id="vpicktree" model="ir.ui.view">
```
Comment (TomohiroAbe-Quartile) [inline stock_picking_ready_flag/views/stock_picking_views.xml:16]: I know the odoo stock_picking_view define this id.
But I think "view_picking_tree" is much better.

You don't need to replace `ref="stock.vpicktree"`, just rename record id.

**Review by TomohiroAbe-Quartile — APPROVED** [review]
Code side: LGTM. Proposing tiny changes.

File: stock_picking_ready_flag/views/stock_picking_views.xml (hunk 2 above)
Comment (tack-yabeQuartile) [reply stock_picking_ready_flag/views/stock_picking_views.xml:16]: Actually odoo/addons/views/stock_picking.xml defines original record of "vpicktree".
In my opinion, changing the record ID does not make much sense, rather forces us to adjust the priority of the view to be displayed.

File: stock_picking_ready_flag/models/__init__.py (hunk 1 above)
Comment (tack-yabeQuartile) [reply stock_picking_ready_flag/models/__init__.py]: OK, I change.

File: stock_picking_ready_flag/views/stock_picking_views.xml (hunk 2 above)
Comment (yostashiro) [reply stock_picking_ready_flag/views/stock_picking_views.xml:16]: `id="vpicktree"` should be fine according to https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#inherited-xml.

**Review by yostashiro — APPROVED** [review]
LGTM 👍

**Comment by tack-yabeQuartile** [comment]
@yostashiro @TomohiroAbe-Quartile 
Thanks your review.  
After test passes, I merge this PR.

**Pull request merged**